contributors/design-proposals/bootstrap-discovery.md

```diff
@@ -101,7 +101,7 @@ An interesting aspect here is that this information is often easily obtained bef
 The user experience for joining a cluster would be something like:
 
 ```
-kubeadm join --token=ae23dc.faddc87f5a5ab458 <address>
+kubeadm join --token=ae23dc.faddc87f5a5ab458 <address>:<port>
```

Comment:
No actually it isn't optional, since it's hard for the community to agree on a default port.
kubeadm uses 6443, GKE 443 IIRC, minikube maybe 8443, etc.